contracts/exchange/test/signature_validator.ts

```diff
@@ -255,185 +224,255 @@ describe('MixinSignatureValidator', () => {
 
         it('should return false when SignatureType=EthSign and signature is invalid', async () => {
             // Create EthSign signature
-            const orderHashHex = orderHashUtils.getOrderHashHex(signedOrder);
-            const orderHashWithEthSignPrefixHex = signatureUtils.addSignedMessagePrefix(orderHashHex);
-            const orderHashWithEthSignPrefixBuffer = ethUtil.toBuffer(orderHashWithEthSignPrefixHex);
-            const ecSignature = ethUtil.ecsign(orderHashWithEthSignPrefixBuffer, signerPrivateKey);
-            // Create 0x signature from EthSign signature
             const signature = Buffer.concat([
-                ethUtil.toBuffer(ecSignature.v),
-                ecSignature.r,
-                ecSignature.s,
-                ethUtil.toBuffer(`0x${SignatureType.EthSign}`),
+                crypto.randomBytes(SIGNATURE_LENGTH),
+                ethUtil.toBuffer([SignatureType.EthSign]),
             ]);
             const signatureHex = ethUtil.bufferToHex(signature);
             // Validate signature.
             // This will fail because `signerAddress` signed the message, but we're passing in `notSignerAddress`
```

Comment:
Small nit: This comment should probably be updated to reflect that `signerAddress` is being passed in, and that the signature is invalid because it's a set of random bytes.
```suggestion
             // This will fail because the `signature` is a sequence of random bytes, so the `signedAddress` 
             // will not be recovered from the signature and the order hash.
```